src/SlackFunction.ts

```diff
@@ -46,83 +113,412 @@ export class SlackFunction {
   private callbackId: string;
 
   /**
-   * @description fn to to process corresponding
+   * @description handler to to process corresponding
    * function_executed event
    */
-  private fn: Middleware<SlackEventMiddlewareArgs>;
+  private handler: Middleware<SlackEventMiddlewareArgs>;
+
+  private interactivityHandlers: FunctionActionMiddleware[];
```

Comment:
How about renaming to FunctionInteractivityMiddleware as this interface is for both action and view?

Reply:
Good catch - I forgot to update the naming after adding in the view method and types. Updated!